.codecov.yml

```diff
@@ -11,6 +11,7 @@ coverage:
     - "src/backend/distributed/utils/citus_outfuncs.c"
     - "src/backend/distributed/deparser/ruleutils_*.c"
     - "src/include/distributed/citus_nodes.h"
+    - "src/backend/distributed/safeclib"
```

Comment:
Why dont we have this in vendor folder?

Reply:
I guess `vendor` might be a default exclude

Reply:
Or actually It might just not have created any coverage files with the `cmake` build

Reply:
I am asking why we dont put safeclib under vendor as that is not our code instead of under src/backend/distributed

Reply:
I added the `vendor` folder in the ignore list as well. Just to be sure now.

And I think I only now understand your question correctly, i.e. why is there code in `src/backend/distributed/safeclib` and not in `vendor`?

The answer to that is: It is actually in `vendor`, `src/backend/distributed/safeclib` is a symlink. The reason why we need the symlink is so that it integrates well with the postgres extension building helpers from Postgres. (see https://github.com/citusdata/citus/pull/3626)